### PR TITLE
fix: Historical task

### PR DESCRIPTION
**Pull request by x-tabdeveloping**

### CODE REVIEW

File: src/seb/registered_models/sentence_transformer_models.py [hunk 1]
```diff
@@ -524,6 +524,25 @@ def create_xlmroberta_en_da_sv_nb() -> SebModel:
...
 
+@models.register("MeMo-BERT-03")
+def create_memo_bert_03() -> SebModel:
+    hf_name = "MiMe-MeMo/MeMo-BERT-03"
+    meta = ModelMeta(
+        name=hf_name.split("/")[-1],
+        huggingface_name=hf_name,
+        reference=f"https://huggingface.co/{hf_name}",
+        open_source=True,
+        languages=["da"],
```
Comment (KennethEnevoldsen) [inline src/seb/registered_models/sentence_transformer_models.py:535]:
```suggestion
        languages=["da", "nb"],
```
Also norwegian right?

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]
Looks good! We should move to the task outside of the main tasks (that or run all models on it).

File: src/seb/registered_models/sentence_transformer_models.py (hunk 1 above)
Comment (x-tabdeveloping) [reply src/seb/registered_models/sentence_transformer_models.py:535]: I just talked to Alie about this and she said that even the Norwegian authors were publishing in Danish, so we should probably keep it this way.

**Review by KennethEnevoldsen — APPROVED** [review]

**Comment by KennethEnevoldsen** [comment]
@x-tabdeveloping seems like tests don't pass (you need to remove an import), but once that is resolved feel free to merge

**Comment by KennethEnevoldsen** [comment]
Fixed the bugs here, will merge

**Pull request merged**